store/src/java/com/zimbra/cs/service/util/VolumeConfigUtil.java

```diff
@@ -100,39 +101,66 @@ public static void validateCreateVolumeRequest(CreateVolumeRequest request, Volu
+                    throw VolumeServiceException.INVALID_REQUEST("Volume UseInFrequentAccess can be TRUE, FALSE or OPTIONAL(don't provide)", VolumeServiceException.BAD_VOLUME_USE_IN_FREQUENT_ACCESS);
+                }
 
-            }
+                // validate use intelligent tiering
+                if (false != volInfoRequest.getVolumeExternalInfo().isUseIntelligentTiering() &&
+                    true != volInfoRequest.getVolumeExternalInfo().isUseIntelligentTiering()) {
+                    throw VolumeServiceException.INVALID_REQUEST("Volume UseIntelligentTiering can be TRUE, FALSE or OPTIONAL(don't provide)", VolumeServiceException.BAD_VOLUME_USE_INTELLIGENT_TIERING);
+                }
 
-            // validate use in frequent access threshold
-            int useInFrequentAccessThreshold = volInfoRequest.getVolumeExternalInfo().getUseInFrequentAccessThreshold();
-            if (0 > useInFrequentAccessThreshold) {
-                throw VolumeServiceException.INVALID_REQUEST("Volume UseInFrequentAccessThreshold can't be negative number", VolumeServiceException.BAD_VOLUME_USE_IN_FREQUENT_ACCESS_THRESHOLD);
-            }
+                // validate use in frequent access threshold
+                int useInFrequentAccessThreshold = volInfoRequest.getVolumeExternalInfo().getUseInFrequentAccessThreshold();
+                if (useInFrequentAccessThreshold < 0) {
+                    throw VolumeServiceException.INVALID_REQUEST("Volume UseInFrequentAccessThreshold can't be negative number", VolumeServiceException.BAD_VOLUME_USE_IN_FREQUENT_ACCESS_THRESHOLD);
+                }
 
-            // validate global bucket id
-            String globalS3BucketId = volInfoRequest.getVolumeExternalInfo().getGlobalBucketConfigurationId();
-            if (false == extVolInfoHandler.validateGlobalBucketID(globalS3BucketId)) {
-                throw VolumeServiceException.INVALID_REQUEST("Volume GlobalBucketID provided is incorrect, missing or empty", VolumeServiceException.BAD_VOLUME_GLOBAL_BUCKET_ID);
-            }
+                // validate global bucket id
+                String globalS3BucketId = volInfoRequest.getVolumeExternalInfo().getGlobalBucketConfigurationId();
+                if (!extVolInfoHandler.validateGlobalBucketID(globalS3BucketId)) {
+                    throw VolumeServiceException.INVALID_REQUEST("Volume GlobalBucketID provided is incorrect, missing or empty", VolumeServiceException.BAD_VOLUME_GLOBAL_BUCKET_ID);
+                }
+                // no validation for volume prefix as of now
+            } else if(AdminConstants.A_VOLUME_OPEN_IO.equalsIgnoreCase(storageTypeOpenIO)) {
+                // validate proxy port as positive
+                if (volInfoRequest.getVolumeExternalOpenIOInfo().getProxyPort() <= 0) {
+                    throw VolumeServiceException.INVALID_REQUEST("Proxy Port can't be negative number or null", VolumeServiceException.BAD_VOLUME_USE_IN_FREQUENT_ACCESS_THRESHOLD);
```

Comment:
make `Port` lower case `port` here and below

Reply:
Fixed in latest commit